src/VisualStudio/VisualBasic/Impl/Options/AdvancedOptionPageControl.xaml.vb

```diff
@@ -115,10 +116,10 @@ Namespace Microsoft.VisualStudio.LanguageServices.VisualBasic.Options
 
             ' Go To Definition
             BindToOption(NavigateToObjectBrowser, VisualStudioNavigationOptions.NavigateToObjectBrowser, LanguageNames.VisualBasic)
-            BindToOption(Enable_all_features_in_opened_files_from_source_generators, SourceGeneratedFileManager.Options.EnableOpeningInWorkspace,
+            BindToOption(Enable_all_features_in_opened_files_from_source_generators, VisualStudioSyntaxTreeConfigurationService.OptionsMetadata.EnableOpeningSourceGeneratedFilesInWorkspace,
```

Comment:
Note that the option does not take effect until VS is restarted.

Reply:
❓ Which part of this option does not take effect immediately?

Reply:
Indeed the code was absolutely supporting that before...I'd hope that wasn't regressed somehow.